### ISSUE TITLE
«Warning: Accessing non-existent property '…' of module exports inside circular dependency»

### ISSUE THREAD

**Issue by maurovanetti** [monaca/monaca-cli#165]
Any monaca command I launch, I get a whole series of warnings about a circular dependency.
The command works perfectly.

For instance, by calling _monaca info_ I get:

```
Monaca dependencies

    monaca-lib      :    ^3.2.10
    monaca-cli      :    3.3.7

System

    os              :    linux
    node            :    15.3.0
    npm             :    7.0.14

Monaca Cloud connection

    local ip        :    192.168.1.79
(node:956) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:956) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:956) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
    cloud status    :    available
…
```

**Comment by yong-asial** [comment]
@maurovanetti Thank you for reporting. This is the known issue. This warning is caused by using Node >= 14.x version in which we, Monaca, haven't supported yet.

**Comment by yong-asial** [comment]
@maurovanetti i have just released the version 3.3.8 to resolve this warnings. Please check it out.
